layers/+email/mu4e/packages.el

```diff
@@ -39,9 +39,28 @@
                 'ivy-completing-read
               'helm-completing-read))
 
+      (when mu4e-async
+        (require 'smtpmail-async)
+        (mu4e/set-send-function 'async-smtpmail-send-it))
+
+      (when (fboundp 'imagemagick-register-types)
+        (imagemagick-register-types))
```

Comment:
I'm surprised to see this low-level function, why do you need to do this ? Are you sure mu4e does not provide a more abstract way to enable display of images ?